assets/textures/CelluloseMembrane.png.import

```diff
@@ -0,0 +1,34 @@
+[remap]
+
+importer="texture"
+type="CompressedTexture2D"
+uid="uid://vl3iystmhirt"
+path="res://.godot/imported/CelluloseMembrane.png-caad8ac6ada9383b24d49ebd8ec265e8.ctex"
+metadata={
+"vram_texture": false
+}
+
+[deps]
+
+source_file="res://assets/textures/CelluloseMembrane.png"
+dest_files=["res://.godot/imported/CelluloseMembrane.png-caad8ac6ada9383b24d49ebd8ec265e8.ctex"]
+
+[params]
+
+compress/mode=0
+compress/high_quality=false
+compress/lossy_quality=0.7
+compress/hdr_compression=1
+compress/normal_map=0
+compress/channel_pack=0
+mipmaps/generate=false
```

Comment:
3D textures should have mipmaps enabled for them, even though this is less critical than for 2D GUI images, I think it is still required ot make sure the texture looks good at small sizes (for example when viewing from far away).

Reply:
Okay. I was trying to get these to match the pre-existing ones, which had mipmaps disabled.